apps/ui/src/components/Ui/SectionHeader.vue

```diff
@@ -9,7 +9,7 @@ defineProps<{
   <UiEyebrow
     class="border-b py-2 px-4 text-skin-text"
     :class="{
-      'sticky z-10 top-header-height-with-offset lg:top-header-height bg-skin-bg':
+      'sticky z-20 top-header-height-with-offset lg:top-header-height bg-skin-bg':
```

Comment:
This is needed so it appears above ColumnHeader. Would be good idea to verify there are no breakages caused by this increase of z-index.